QemuPkg/Binaries/qemu-windows_ext_dep.yaml

```diff
@@ -6,7 +6,7 @@
 # SPDX-License-Identifier: BSD-2-Clause-Patent
 ##
 {
-  "scope": "global-win",
+  "scope": "glssdobal-win",

```

Comment:
@Javagedes I observed some issues when launching qemu-aarch64 from windows for missing bosch binary. Can you send me a pointer on where I can fix it?

Reply:
do you mean this? https://github.com/bochs-emu/Bochs

I don't see any reference to bosch / bochs in the qemu folder from the normal qemu installation (C:\Program Files\qemu).

Was this something you manually installed locally?

If it something that comes bundled with qemu, and I just can't find it - then we just need to add it to the extdep release workflow here:
https://github.com/microsoft/mu_tiano_platforms/blob/871d48020bad9482979905db7b1116c2f81573a6/.github/workflows/publish-qemu-bin.yml#L40****

If it is another tool, we can probably bundle it together in the qemu extdep so that it is available.